projects/power_distribution/inc/pd_fan_ctrl.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+// Control fans for front and rear power distro via ADT7476A
+
+#include "pd_fan_ctrl_defs.h"
+#include "pin_defs.h"
+#include "soft_timer.h"
+#include "status.h"
+
+typedef struct FanCtrlStorage {
+  GpioAddress enc_vent_therm_pin;
+  GpioAddress dcdc_therm_pin;
+  GpioAddress pot_pin;
+  I2CPort i2c;
```

Comment:
Nitpick: this is somewhat cryptic, it's not immediately clear if it refers to the I2C port or address. I'd call it `i2c_port`, same for in the settings struct.